Fix error where TwainScanner cleans out folders

Rekperion opened the pull request:
This fixes an issue discovered in Financial where the scanner was taking out scanned files that are within the "OCR" folder which is considered as a "custom folder" to delete everything out of. This Will probably require a follow up fix to allow for proper cleaning of temp items from folders.

Rekperion committed:
Fix error where TwainScanner cleans out folders

This fixes an issue discovered in Financial where the scanner was taking out scanned files that are within the "OCR" folder which is considered as a "custom folder" to delete everything out of. This Will probably require a follow up fix to allow for proper cleaning of temp items from folders.

majorsilence commented:
> This fixes an issue discovered in Financial where the scanner was taking out scanned files that are within the "OCR" folder which is considered as a "custom folder" to delete everything out of. This Will probably require a follow up fix to allow for proper cleaning of temp items from folders.

I'm going to need more details.   The ocr folder is a temp folder where files are not supposed to be kept once saved.

Rekperion committed:
Changed to work as old functionality

Changed how the DeleteFiles() method works. Now only deletes the files that the TwainScanner project itself would generate. Instead of all files in a custom folder or all tmpscan files in the temp folder.

majorsilence commented on TownSuite.TwainScanner/MainFrame.cs:
```diff
@@ -131,27 +131,12 @@ private void MainFrame_Load(object sender, EventArgs e)
-                LoopFiles("tmpscan*.pdf");
-                LoopFiles("tmpscan*.txt");
-            }
+            LoopFiles("tmpScan.bmp");
+            LoopFiles("tmpScan.jpeg");
+            LoopFiles("tmpScan.tif");
+            LoopFiles("tmpScan.png");
+            LoopFiles("tmpScan.pdf");
+            LoopFiles("tmpScan.txt");
```
This leaves temp files hanging around on the file system

![image](https://github.com/TownSuite/TownSuite.TwainScanner/assets/656288/14a56363-1d98-413a-afe3-ef289d9bd8cd)
